include/solvers/gls_sharp_navier_stokes.h

```diff
@@ -159,6 +159,15 @@ class GLSSharpNavierStokesSolver : public GLSNavierStokesSolver<dim>
   void
   refine_ib();
 
+  /**
+   * @brief
+   *This function create a map with the key being the cell and what it contains
+   *a bool to define if a cell is cut and the particle that cut it if it's the
+   *case. This map contains is referred as cutted_cells_map.
```

Comment:
```suggestion
   *This function create a map (cut_cells_map) that indicates if a cell is cut.
```
cut in the past is cut, not cutted